src/test/java/org/example/AppTest.java

```diff
@@ -31,4 +38,17 @@ public static Test suite() {
     public void testApp() {
         assertTrue( true );
     }
+
+    public void testList(){
+        String[] data = {"one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"};
+        String[] expected = {"eight", "five", "four", "nine", "one", "seven", "six", "ten", "three", "two"};
+        Object[] actual = App.sorted_list(data);
+
+        Iterator<String> ex = Arrays.stream(expected).iterator();
+        Iterator<Object> ac = Arrays.stream(actual).iterator();
+        while(ex.hasNext() && ac.hasNext()){
```

Comment:
I propose to use to update junit library to the 5 version and use this for comparing lists/arrays https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html